### PR TITLE
A TON of userrights configuration for usawiki (#2)

### PR DESCRIPTION
**Pull request by AgraveBanks**

### CODE REVIEW

File: LocalWiki.php [hunk 1]
```diff
@@ -232,6 +232,60 @@ function onClosedSiteNoticeAfter( &$siteNotice, $skin ) {
...
+	$wgGroupPermissions['sysop']['deletedtext'] = false;
+	$wgGroupPermissions['sysop']['move-categorypages'] = false;
+	$wgGroupPermissions['sysop']['movefile'] = false;
+	$wgGroupPermissions['sysop']['move'] = false;
+	$wgGroupPermissions['sysop']['move-subpages'] = false;
+	$wgGroupPermissions['sysop']['move-rootuserpages'] = false;
+	$wgGroupPermissions['sysop']['abusefilter-modify'] = false;
+	$wgGroupPermissions['sysop']['abusefilter-revert'] = false;
+	$wgGroupPermissions['sysop']['abusefilter-modify-restricted'] = false;
+	$wgGroupPermissions['sysop']['abusefilter-private'] = false;
```
Comment (The-Voidwalker) [inline LocalWiki.php]: you can't use `abusefilter-private`, it provides access to user's IPs

**Comment by AgraveBanks** [comment]
@The-Voidwalker which userright allows viewing of abuse filters marked as private? I know that `abusefilter-view` grants access to Special:AbuseFilter altogether, but isn't there a separate permission that is required to view private filters?

**Comment by The-Voidwalker** [comment]
`abusefilter-view-private` and `abusefilter-log-private`

**Comment by labster** [comment]
PR by a ban evader. Closing PR.

**Pull request closed without merging**